physics/radiation_clouds.f

```diff
@@ -2216,8 +2216,10 @@ subroutine progcld_thompson_wsm6                                  &
 
         do k = 1, NLAY-1
           do i = 1, IX
-            cwp(i,k) = max(0.0, clw(i,k,ntcw) * gfac * delp(i,k))
-            cip(i,k) = max(0.0, clw(i,k,ntiw) * gfac * delp(i,k))
+            cwp(i,k) = max(0.0, (clw(i,k,ntcw)+cnvw(i,k)*
```

Comment:
I recommend documenting this code change at Line 2215: "...considering the impact of convective clouds."

Reply:
@mzhangw Great idea. Done